go.mod

```diff
@@ -422,3 +457,6 @@ replace (
 	// TODO(vladklokun): armoapi-go requires gorm v1.24.6, but Grype depends on v1.23.10 and breaks on anything above. We can’t have both. Upstream changes to Grype?
 	gorm.io/gorm => gorm.io/gorm v1.23.10 // indirect
 )
+
+// TODO(anubhav06): Remove this once we have a release of copacetic with the support for patching kubescape image scan results.
```

Comment:
@Daniel-GrunbergerCA @anubhav06 please remove this replace

Reply:
The integration between copa and kubescape is not live yet. For the meantime, I have added the integration between copa and kubescape in a fork of mine(of copa).
I'm actively working with the copa team on the feature of incorporating multiple scanners in copa. Once we have a release of copa which supports kubescape, we can remove this.

Reply:
I've never seen the license check pass in the GitHub Action.  Someone should fix this, but we also have FOSSA, which says everythign is OK.

Reply:
I've also made the [PR#1374](https://github.com/kubescape/kubescape/pull/1374) which changes reference back to copa, and in that version copa has Apache-2.0-License (previously MIT), incase that helps in fixing the check.